wpapi-class-shortcode.php

```diff
@@ -111,18 +111,18 @@ function get_api($attr){
             'type' => 'posts',
             'size' => 'medium',
         ), $attr));
-        $url = "{$url}/wp-json/{$type}?{$q}";
+        $url = "{$url}/wp-json/wp/v2/{$type}?_embed{$q}";
         $wp_api_posts = wp_remote_get($url);
         $html = "<ul class='wpapi wpapi-shortcode'>";
 
         $WpapiContent = new WpapiContents();
-        if(is_wp_error($wp_api_posts))}
+        if(is_wp_error($wp_api_posts)){
             $html = "<dl><dt>faild get WP-API</dt></dl>";
             return $html . "</ul>";
         } elseif($wp_api_posts['response']['code'] != 200){
             return $html . $WpapiContent->get_badresponse($wp_api_posts);
         }
-        $wp_api_posts = json_decode($wp_api_posts['body']);
+		$wp_api_posts = json_decode( json_encode( json_decode($wp_api_posts['body']) ),true );

```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* No space found after comma in function call